### PR TITLE
Feature/user can get names of all spark jobs/#63

### PR DESCRIPTION
**Pull request by oliver-tarrant-tessella**
New `getjobs` handler that returns a list of spark applications with their status. To be addressed in a later pull request will be the ability to filter by a label see #64 

## Stories covered:
#63 

## To test
Run system tests

### CODE REVIEW

File: piezo_web_app/PiezoWebApp/src/handlers/get_jobs.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# pylint: disable=abstract-method
+class GetJobsHandler(BaseHandler):
+    @schema.validate(
+        output_schema={
+            "type": "object",
+            "properties": {
+                "message": {"type": "string"},
+            },
+        }
```
Comment (robert-clegg-tessella) [inline piezo_web_app/PiezoWebApp/src/handlers/get_jobs.py]: This doesn't match up with what should be returned (missing `"spark_jobs"`, I think)

File: piezo_web_app/PiezoWebApp/tests/handlers/get_jobs_test.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+from mock import call
+from tornado.testing import gen_test
+
+from PiezoWebApp.tests.handlers.base_handler_test import BaseHandlerTest
+from PiezoWebApp.src.handlers.get_jobs import GetJobsHandler
+
+
+class TestGetLogsHandler(BaseHandlerTest):
```
Comment (robert-clegg-tessella) [inline piezo_web_app/PiezoWebApp/tests/handlers/get_jobs_test.py]: Class name is wrong

Comment (oliver-tarrant-tessella) [reply piezo_web_app/PiezoWebApp/tests/handlers/get_jobs_test.py]: > Looks good, but the [Web App User Guide](https://github.com/ukaea/piezo/wiki/WebAppUserGuide) also needs updating

https://github.com/ukaea/piezo/wiki/WebAppUserGuide#get-list-of-jobs

File: piezo_web_app/PiezoWebApp/src/handlers/get_jobs.py (hunk 1 above)
Comment (oliver-tarrant-tessella) [reply piezo_web_app/PiezoWebApp/src/handlers/get_jobs.py]: 2918bdd

File: piezo_web_app/PiezoWebApp/tests/handlers/get_jobs_test.py (hunk 2 above)
Comment (oliver-tarrant-tessella) [reply piezo_web_app/PiezoWebApp/tests/handlers/get_jobs_test.py]: f298795

**Pull request merged**